akka-http-core/src/main/java/akka/http/javadsl/model/HttpMethod.java

```diff
@@ -42,14 +42,13 @@ public final String name() {
 
     /**
      * Returns the entity acceptance level for this method.
-     * @deprecated Use {@link #getRequestEntityAcceptance} instead, which returns {@link akka.http.javadsl.model.RequestEntityAcceptance}.
      */
-    @Deprecated
-    public abstract akka.http.scaladsl.model.RequestEntityAcceptance requestEntityAcceptance();
+    public abstract akka.http.javadsl.model.RequestEntityAcceptance requestEntityAcceptance();
```

Comment:
Subtle - so if you'd want to support both akka-http 10.0 and 10.1 you'd have to use the now-deprecated `getRequestEntityAcceptance` - that makes sense and is in line with the comment on the 'old' `requestEntityAcceptance()`, so 👍